edison-test/index-test.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE html>
```

Comment:
Just by looking at the name, it seems like this file was created for testing purposes. If thats true, lets not check it in.